README.md

```diff
@@ -25,6 +25,15 @@ You can download the .NET SDK as either an installer (MSI, PKG) or a zip (zip, t
 > [!NOTE]
 > When acquiring installers from the latest builds table, be aware that the installers are the **latest bits**. With development builds, internal NuGet feeds are necessary for some scenarios (for example, to acquire the runtime pack for self-contained apps). You can use the following NuGet.config to configure these feeds. See the following document [Configuring NuGet behavior](https://docs.microsoft.com/nuget/consume-packages/configuring-nuget-behavior) for more information on where to modify your NuGet.config to apply the changes.
 
+### For .NET 10 builds
+```xml
+<configuration>
+  <packageSources>
+    <add key="dotnet9" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnet10/nuget/v3/index.json" />
+  </packageSources>
+</configuration>
+```
+
```

Comment:
Just a suggestion: To make the package source configuration more flexible and easier to update for future versions, we could use x as a placeholder for the .NET version number.
```
<configuration>
  <packageSources>
    <add key="dotnetx" value="https://pkgs.dev.azure.com/dnceng/public/_packaging/dotnetx/nuget/v3/index.json" />
  </packageSources>
</configuration>

Usage
Replace x with the specific version number (e.g., 8, 9, 10, etc.).
```

Reply:
Certainly an interesting idea that would reduce future costs. However, I know I often will copy directly from this page and don't want to have to take an additional step of updating a placeholder value. I assume others as well just want to be able to copy paste as well so I'm hesitant to change this to match that. We could do a vNext (dotnet10) and a Generic one (dotnetN) but not sure there's enough benefit.

Reply:
Is it supposed to say net10 in the URL since the key says net9?